examples/ecdsa-psbt.rs

```diff
@@ -319,7 +297,7 @@ enum Error {
     /// PSBT error.
     Psbt(psbt::Error),
     /// PSBT sighash error.
-    PsbtSighash(SighashError),
+    PsbtSign(psbt::SignError),
```

Comment:
How would we handle errors then, using `unwrap` or `expect` everywhere?

Reply:
My thinking was examples are full of unwraps/expects.

Reply:
Fair point. I thought this example was so complex that error handling was useful. Is there other any reason to remove it now its there?

Reply:
I am of the point that examples should be short/simple to illustrate the good case/happy path of the API. People can figure out error handling based on their own custom requirements. 

Almost all the doctests I have read do not touch error handling. They just explain the API with minimal code. I won't oppose having it in this example if you feel it's okay to keep it

Reply:
I hear your point. In this case I think it makes the code more terse since we can use `?` everywhere. Agreed the actual error type and boiler plate are a little over the top but since we don't want any dev dependencies I don't see a better way. Its only 50 LOC and folks will likely be used to our error boilerplate if they have looked at the code at all.

Reply:
Just have `struct Error(Box<std::error::Error>)` and `impl<T: std::error::Error> From<T> for Error` + `impl Debug for Error` by displaying the error with sources to enable returning it from `main`

Reply:
You the man.